grunttasks/bump.js

```diff
@@ -9,11 +9,11 @@
 module.exports = function (grunt) {
   grunt.config('bump', {
     options: {
-      files: ['package.json', 'npm-shrinkwrap.json'],
+      files: ['package.json', 'fxa-oauth-server/package.json', 'npm-shrinkwrap.json'],
```

Comment:
Shouldn't we bump both shrinkwraps too?

Reply:
Yeah, you're right. `__version__` looks at `package.json`, so solves the issue at hand, but bumping both is correct. I'll update.